run-tests.pl

```diff
@@ -737,8 +737,9 @@ sub list_symbols
 
          no warnings 'once';
 
-         local *test       = sub { _push_test( $filename, 0, @_ ); };
-         local *multi_test = sub { _push_test( $filename, 1, @_ ); };
+         local *test        = sub { _push_test( $filename, 0, @_ ); };
+         local *multi_test  = sub { _push_test( $filename, 1, @_ ); };
+         local *broken_test = sub {};
```

Comment:
well, if we're going to do this, then let's document what it's for, both here and in DEVELOP.rst. But I'm a bit unconvinced - the `bug` param means we'll expect it to fail, so why isn't that good enough?

Reply:
stick a timeout on sync_until_user_in_device_list if necessary.